packages/cfd/src/Components/cfd-real-account-display.tsx

```diff
@@ -148,7 +148,11 @@ const CFDRealAccountDisplay = ({
 
         if (is_eu) {
             const account = getAccountTypeFields({ category: 'real', type: sub_account_type });
-            return isAccountOfTypeDisabled(account?.account_type);
+            return isAccountOfTypeDisabled(
+                account?.account_type as unknown as DetailsOfEachMT5Loginid[] & {
```

Comment:
Type casting makes TS to not make any type check and it is not suggested. Can you come up with another solution ?

Reply:
The thing is that this account_type has two different type in it thats why i used this type casting will have to refactor the components to make individual types. Thats why for time being it is used to handle any app crash.

Reply:
Kindly help to add a comment here so that we do not miss it later